Doc/Manual/Php.html

```diff
@@ -821,6 +821,15 @@
 </pre></div>
 
 <p>
+The <b>version</b> pragma can be used to add version to PHP module in the generated "example.php". The version is inserted in the zend_module_entry block.
```

Comment:
That's not quite right - it adds a version to the generated **PHP extension** (not the generated PHP wrapper script).